core/src/main/java/org/mule/routing/filters/ExpressionFilter.java

```diff
@@ -124,6 +111,34 @@ public boolean accept(MuleMessage message)
         }
     }
 
+    /**
+     * Check a given event against this filter.
+     *
+     * @param event a non null event to filter.
+     * @return <code>true</code> if the event matches the filter
+     */
+    @Override
+    public boolean accept(MuleEvent event)
+    {
+        // MULE-4797 Because there is no way to specify the class-loader that script
```

Comment:
Is code was copied/pasted from the accept(MuleMessage message) method. I would like you to remove that duplication and ask you why there is still code for that other accept method?